### PR TITLE
Just add the code of conduct as requested.

### PR DESCRIPTION
**Pull request by namanrai**
added the code of conduct as requested for the task.
this is my first gci task for this year.
help me anywhere if i am wrong

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `grunt jasmine`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] PR body includes `fixes #0000`-style reference to original issue #
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

**Comment by namanrai** [comment]
@publiclab/reviewer help me if i am wrong somewhere

File: README.md [hunk 1]
```diff
@@ -2,7 +2,9 @@ PublicLab.Editor
 ====
 
 [![Build Status](https://travis-ci.org/publiclab/PublicLab.Editor.svg)](https://travis-ci.org/publiclab/PublicLab.Editor)
-
+[![Code of Conduct](https://img.shields.io/badge/
+code-of%20conduct-green.svg)]
```
Comment (keshav234156) [inline README.md]: Can you please merge line 5 and 6 into a single line

**Comment by namanrai** [comment]
How do I merge these lines?

**Comment by keshav234156** [comment]
Thanks @namanrai for you amazing work

**Comment by keshav234156** [comment]
@jywarren can you please review the changes.

**Comment by namanrai** [comment]
Thanks alot!

**Comment by SidharthBansal** [comment]
Thanks for your hard work 💯 
Please visit https://github.com/publiclab/plots2/issues/6808 🥇 
Also, please install a repository for PL. @keshav234156  will guide you in this regard
Please provide a screenshot of the badge in the readme. 
Superb work 🎉

**Comment by LeoDog896** [comment]
Seems fine! Just make sure the title and description are more professional; other than that, looks good!

**Comment by jywarren** [comment]
Awesome. Thanks a lot!!! 🎉

**Pull request merged**